indico/modules/events/editing/controllers/backend/timeline.py

```diff
@@ -128,8 +135,19 @@ def _process(self):
         args = parser.parse({
             'files': EditingFilesField(self.event, self.contrib, self.editable_type, required=True)
         })
-
-        create_new_editable(self.contrib, self.editable_type, session.user, args['files'])
+        service_url = editing_settings.get(self.event, 'service_url')
+        initial_state = InitialRevisionState.new if service_url else InitialRevisionState.ready_for_review
+
+        editable = create_new_editable(self.contrib, self.editable_type, session.user, args['files'], initial_state)
+        db.session.commit()
+        if service_url:
+            try:
+                service_handle_editable(editable)
+            except ServiceRequestFailed as exc:
+                delete_editable(editable)
+                db.session.commit()
+                logger.exception('Failed to contact service. Got: %r', exc.message)
+                raise BadRequest(_('Failed while submitting for editing. The associated service is not available.'))
```

Comment:
Sounds a bit strange. What about something more generic like "Submission failed, please try again later."? The end user does not really know about the microservice etc.

Reply:
It is a good idea but might induce into the mistake of thinking the submission error was related to the editable itself, instead of the service that processes it.

Reply:
Not really important for the user, and I think "try again later" indicates that it's not something the user did wrong. In any case, the service should be stable ;)